Sources/Workspace/DefaultPluginScriptRunner.swift

```diff
@@ -26,15 +26,17 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner, Cancellable {
     private let toolchain: UserToolchain
     private let enableSandbox: Bool
     private let cancellator: Cancellator
+    private let extraPluginFlags: [String]
```

Comment:
alternatively, we can pass `isVerbose` so this is more structured / less open ended. 

do you have an opinion on what the right level of abstraction is the right one in this case? 

cc @abertelrud

Reply:
I was following the same approach as for manifests here -- I think eventually it would make sense to also have a `-Xcplugins` to pass additional options that why this made sense to me.

Reply:
That does seem reasonably, especially as people try out new compilers and toolchains with existing SwiftPM builds.  I think verbosity is common enough to warrant a boolean, so I can see having an `isVerbose` for now but adding `Xplugins` for additional flags.